pkgs/development/python-modules/geopandas/default.nix

```
@@ -39,10 +50,30 @@ buildPythonPackage rec {
     shapely
   ];
 
+  passthru.optional-dependencies = {
```

```suggestion
  optional-dependencies = {
```
See https://github.com/NixOS/nixpkgs/blob/1d89edf78b0f65e76e3df1f32be6f7d4c5afea76/pkgs/development/interpreters/python/mk-python-derivation.nix#L229

---

Thank you, done!